ksqldb-rest-app/src/test/java/io/confluent/ksql/api/ApiTest.java

```diff
@@ -331,7 +331,7 @@ public void shouldHandleErrorInProcessingQuery() throws Exception {
     assertThat(queryResponse.rows, hasSize(DEFAULT_JSON_ROWS.size() - 1));
     validateError(ERROR_CODE_SERVER_ERROR, "java.lang.RuntimeException: Failure in processing", queryResponse.error);
     assertThat(testEndpoints.getPublishers(), hasSize(1));
```

Comment:
This line conflicted. `7.3.x` code uses `getQueryPublishers()` instead of `getPublishers()`